plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1964,6 +1969,473 @@ public void testQueryTableFunction()
                 .failure().hasMessageContaining("json_parse_exception");
     }
 
+    @Test
+    public void testSimpleProjectionPushdown()
```

Comment:
https://trino.io/docs/current/develop/tests.html#conventions-and-recommendations

> Test methods should be defined as package-private.